.travis.yml

```diff
@@ -3,12 +3,12 @@ python: 3.8
 cache: pip
 
 before_install:
-- pip install -r requirements/all.requirements
+- pip install -r ../collector/requirements
```

Comment:
It's a wrong path to requirements. And is it better to name it `requirements.txt`, isn't it?